### PR TITLE
Update comments to Ubuntu 20.04

### PR DESCRIPTION
**Pull request by Urhengulas**
We use use Ubuntu 20.04 instead of 18.04 since https://github.com/ferrocene/ferrocene/pull/376.

### CODE REVIEW

**Comment by tshepang** [comment]
I considered doing this change, but paused because I was not sure the comments were all true for 20.04. Can you confirm?

**Comment by pietroalbini** [comment]
They are all true.

**Review by Hoverbear — APPROVED** [review]

**Review by tshepang — APPROVED** [review]
bors merge

File: ferrocene/ci/docker-images/ubuntu-20/Dockerfile [hunk 1]
```diff
@@ -102,15 +102,15 @@ RUN mkdir /opt/nodejs \
 ENV PATH "/opt/nodejs/bin:$PATH"
 
 # Install AWS CLI from the published binaries, as no source tarball is provided
-# and the version in the Ubuntu 18.04 archive is too old and doesn't support
+# and the version in the Ubuntu 20.04 archive is too old and doesn't support
 # `aws ecr get-login-password`.
 RUN curl -Lo /tmp/awscli.zip https://awscli.amazonaws.com/awscli-exe-linux-x86_64-2.12.4.zip \
```
Comment (tshepang) [inline ferrocene/ci/docker-images/ubuntu-20/Dockerfile:107]: `get-login-password` is in a version available from 20.04... maybe we can try that in a future pr

Comment (pietroalbini) [reply ferrocene/ci/docker-images/ubuntu-20/Dockerfile:107]: Ubuntu 20.04 still has AWS CLI 1.x, we need 2.x.

**Pull request merged**